commands/root.go

```diff
@@ -38,7 +38,7 @@ var (
 		Short: "Hazelcast command-line client",
 		Long:  "Hazelcast command-line client connects your command-line to a Hazelcast cluster",
 		Example: "`hzc map --name my-map put --key hello --value world` - put entry into map directly\n" +
-			"`hzc --help` - print help",
+			"`hzc help` - print help",
```

Comment:
`hzc --help` and `hzc help` seem to return the same thing. Did you prefer the latter since it's simpler?

Reply:
Yes also it is mentioned as a command in our help text as 'hzc help'